polyn_elixir_client/README.md

```diff
@@ -122,14 +114,35 @@ Polyn uses a shared Key-Value bucket in NATS to avoid re-running migrations. It
 
 When using `mix release` to deploy, `mix` and Mix Tasks are not available, so you can't use `mix polyn.migrate` to do your migrations.
 
-Instead you can use a built-in `Polyn.Release` module to execute migrations in the compiled application
+Instead you'll need to run `mix polyn.gen.release` which will add a `lib/my_app/release.ex` file to your app (if you already have this file it will append to it). The file will look something like this:
+
+```elixir
+defmodule MyApp.Release do
+  @app :my_app
+
+  def polyn_migrate do
+    load_app()
+    {:ok, _apps} = Application.ensure_all_started(:polyn)
 
-You can use it from the release like this:
+    dir = Path.join([:code.priv_dir(@app), "polyn", "migrations"])
+    Polyn.Migration.Migrator.run(migrations_dir: dir)
```

Comment:
❤️ 💛 💙 💚 💜 I love that the API has become as simple as "run all the migrations in this directory". This simple out-of-the-box process will be easy for people to adopt, but also if they want to apply migrations after boot, or some other interesting use-case, they just need to find their directory and call this function 🙌